components/taggroup/stories/taggroup.stories.js

```diff
@@ -37,6 +37,14 @@ export default {
 			},
 			control: "boolean",
 		},
+		customStyles: {
```

Comment:
Should this be hidden from the UI? That lets us pass data to it but not show the property to users so they don't think it's part of the formal API?

Reply:
I put it under a separate category "Storybook Only", with the thinking that users may want to see what extra styles are being applied that cause the story to display differently, and/or may want to test a different value (changing max-width for example I found useful with testing Table). And it only displays if that arg is set. Happy to hide it though if you think it'd be better not to show in the UI.

Reply:
That makes sense! No changes needed.